sdk/storage/Azure.Storage.sln

```diff
@@ -111,8 +111,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Batch", "Batch", "{610DBA57
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Storage.Blobs.Batch.Tests", "Azure.Storage.Blobs.Batch\tests\Azure.Storage.Blobs.Batch.Tests.csproj", "{2C48AA18-94E2-4B8E-9345-DCA941B13EB1}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ApiCompat", "..\..\eng\ApiCompat\ApiCompat.csproj", "{16260507-C87E-44D2-883F-8E338B7A519F}"
```

Comment:
Was this intentional?

Reply:
Yep, otherwise it doesn't build.  @JoshLove-msft told me to do this.

Reply:
This was just a workaround as it wasn't building for you locally on the 71 branch. We don't want to check this in though since it would mean we won't have automatic restores for ApiCompat.

Reply:
I can't build the master branch without removing this line.

Reply:
Fixed.